client/my-sites/checklist/wpcom-checklist/checklist-prompt/index.jsx

```diff
@@ -31,31 +32,31 @@ export class ChecklistPrompt extends Component {
 	};
 
 	render() {
-		const { translate, taskList, children } = this.props;
+		const { translate, taskList, children, promptTaskId } = this.props;
 
 		const childrenArray = Children.toArray( children );
 
 		const firstIncomplete = taskList.getFirstIncompleteTask();
 		const isFinished = ! firstIncomplete;
+		const promptTask = promptTaskId && taskList.get( promptTaskId );
+		const theTaskId = promptTask ? promptTask.id : firstIncomplete.id;
 
 		return (
 			<div className="checklist-prompt">
 				{ isFinished ? (
-					<>
-						<ChecklistPromptTask
-							description={ translate(
-								'We did it! You have completed {{a}}all the tasks{{/a}} on our checklist.',
-								{
-									components: {
-										a: <a href={ `/checklist/${ this.props.siteSlug }` } />,
-									},
-								}
-							) }
-							title={ translate( 'Your site is ready to share' ) }
-						/>
-					</>
+					<ChecklistPromptTask
+						description={ translate(
+							'We did it! You have completed {{a}}all the tasks{{/a}} on your checklist.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'We did it! You have completed {{a}}all the tasks{{/a}} on our checklist.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).